portal/app/[locale]/_components/navbar/_components/hemiLogo.tsx

```diff
@@ -1,12 +1,19 @@
 export const HemiLogoFull = () => (
-  <svg fill="none" height={24} width={70} xmlns="http://www.w3.org/2000/svg">
-    <path
-      d="M29.089 5.259v5.787l-.29-.097c.193-.839.587-1.452 1.18-1.839.594-.4 1.303-.6 2.13-.6 1.057 0 1.896.355 2.515 1.065.632.697.948 1.638.948 2.825V19h-2.051v-5.864c0-.942-.161-1.645-.484-2.11-.322-.464-.826-.696-1.51-.696-.748 0-1.341.245-1.78.735-.439.477-.658 1.168-.658 2.07V19h-2.051V5.259h2.05Zm8.131 8.612c0-1.07.2-2.013.6-2.825.413-.813.987-1.439 1.723-1.878.735-.438 1.593-.658 2.574-.658.838 0 1.6.194 2.284.581.696.387 1.251.98 1.664 1.78.413.787.639 1.762.677 2.923l.02.658h-7.374c.064.955.335 1.703.813 2.245.477.529 1.116.793 1.916.793.516 0 .98-.141 1.393-.425a2.55 2.55 0 0 0 .949-1.162l2.148.155a4.308 4.308 0 0 1-1.684 2.323c-.813.567-1.748.851-2.806.851-.98 0-1.839-.22-2.574-.658-.736-.439-1.31-1.064-1.723-1.877-.4-.813-.6-1.755-.6-2.826Zm7.374-.948c-.116-.916-.4-1.587-.852-2.013-.438-.438-.98-.658-1.625-.658-.749 0-1.355.239-1.82.716-.464.465-.754 1.116-.87 1.955h5.167Zm5.83-4.18.038 1.683c.246-.606.62-1.077 1.123-1.412.516-.336 1.116-.504 1.8-.504.735 0 1.361.187 1.877.562.516.374.884.896 1.103 1.567.232-.696.607-1.225 1.123-1.587.529-.361 1.18-.542 1.955-.542.683 0 1.27.155 1.76.465.504.297.884.735 1.142 1.316.272.58.407 1.284.407 2.11V19H60.7v-5.96c0-.904-.154-1.594-.464-2.072-.31-.49-.755-.735-1.335-.735-.478 0-.89.11-1.239.329-.335.22-.593.542-.774.968-.18.412-.271.916-.271 1.51V19h-1.935v-5.96c0-.904-.162-1.594-.484-2.072-.31-.49-.749-.735-1.316-.735-.478 0-.89.11-1.239.329-.336.22-.593.542-.774.968-.18.412-.271.916-.271 1.51V19h-2.052V8.743h1.878Zm16.91 0V19h-2.051V8.743h2.051Zm.058-3.465v1.994h-2.148V5.278h2.148Z"
-      fill="#000202"
-    />
-    <path
-      d="M12.09 2.019a.19.19 0 0 0-.222.155L10.61 9.402h-.94L8.412 2.174a.19.19 0 0 0-.221-.155C3.72 2.895.295 6.748.012 11.459c0 .006-.012.205-.012.304V12.093c0 4.996 3.524 9.154 8.197 10.073a.19.19 0 0 0 .22-.155l1.26-7.228h.939l1.252 7.234a.19.19 0 0 0 .221.156c4.47-.883 7.89-4.736 8.179-9.447 0-.006.012-.205.012-.304V12.092c.006-4.996-3.518-9.154-8.19-10.073Z"
-      fill="#FF6C15"
-    />
+  <svg fill="none" height="24" width="71" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#hemiLogoFullClipPath)">
```

Comment:
```suggestion
    <g clipPath="url(#hemiLogoFullClipPath)">
```

<img width="1162" height="144" alt="Image" src="https://github.com/user-attachments/assets/43c0a5fa-fa50-4d58-a77e-89af38aff184" />